css/at-rules/media.json

```diff
@@ -1188,6 +1188,55 @@
             }
           }
         },
+        "prefers-color-scheme": {
+          "__compat": {
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@media/prefers-color-scheme",
+            "description": "<code>prefers-color-scheme</code> media feature",
+            "support": {
+              "chrome": {
+                "version_added": false
```

Comment:
74 for all 3 flavors. https://storage.googleapis.com/chromium-find-releases-static/795.html#7959d13749d4d965ad6a08767e3d28dabe262d03

Reply:
Not quite. [7959d137](https://chromium.googlesource.com/chromium/src/+/7959d13749d4d965ad6a08767e3d28dabe262d03) wires it up in Blink, but currently users can't change it from the default – `no-preference`.